### PR TITLE
Add OHF logo to README

### PR DESCRIPTION
**Pull request by mrdarrengriffin**
## Proposed change

Adds the OHF logo to the README

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added a badge linking to the Open Home Foundation website in the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +40,5 @@ When you're done working on the site, run the following command to move the post
 ```bash
 bundle exec rake integrate
 ```
+##
```
Comment (frenck) [inline README.md]: Why the `##`?

File: README.md [hunk 2]
```diff
@@ -40,3 +40,5 @@ When you're done working on the site, run the following command to move the post
...
+[<img src="https://www.openhomefoundation.org/badges/home-assistant.png">](https://www.openhomefoundation.org/)
```
Comment (frenck) [inline README.md]: We can avoid the use of HTML here, right?

**Review by frenck — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mrdarrengriffin) [reply README.md]: Just to add a separator but we can leave out if you'd prefer

File: README.md (hunk 2 above)
Comment (mrdarrengriffin) [reply README.md]: Ah yes we can now. It was markdown previously but we needed to force a specific size. I will amend

File: README.md (hunk 1 above)
Comment (mrdarrengriffin) [reply README.md]: Removed

**Review by frenck — APPROVED** [review]
Thanks, @mrdarrengriffin 👍 

../Frenck

**Pull request merged**